CONFIGURE.md

```diff
@@ -938,7 +943,15 @@ To enable FIPS support, and use MiNiFi C++ in FIPS compliant mode, there are a f
     # in minifi.properties
     nifi.openssl.fips.support.enable=true
 
-Before first starting the application, the fipsmodule.cnf needs to be generated. To do this run the following command with the openssl binary (openssl on Unix and openssl.exe on windows) with the following parameters provided in the $MINIFI_HOME/fips directory:
+Before first starting the application, the fipsmodule.cnf needs to be generated. This can be done in two ways, either automatically or manually.
+
+#### Generating the fipsmodule.cnf file automatically
+
+If the application is started with the nifi.openssl.fips.support.enable property set to true, and the fipsmodule.cnf file is not found in the $MINIFI_HOME/fips directory, the application will try to generate the fipsmodule.cnf file automatically. This is done by running the manual steps described in the next section, but this is done from the MiNiFi C++ process before loading the OpenSSL configuration. If the automatic generation is successful, the application will start in FIPS mode.
+
+#### Generating the fipsmodule.cnf file manually
```

Comment:
Can we just remove the manual steps, or is there an advantage of doing it manually in some cases?